schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,54 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  event:
+    type: object
+    properties:
+      uri:
+        type: string
+        format: uri
+        description: the unique URI identifying the event
+      request_id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique UUID v1 ID of the event derived from the X-Request-Id header
+      id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique ID of this event; should match the ts field
+      ts:
+        type: string
+        format: date-time
+        description: 'the time stamp of the event, in ISO8601 format'
+      domain:
+        type: string
+        description: the domain the event pertains to
+    required:
+      - uri
+      - request_id
+      - id
+      - ts
+      - domain
+  # event-specific fields
+  title:
+    type: string
+    description: the title of the page
+  page_id:
+    type: integer
+    minimum: 1
+    description: the page ID of the deleted page
+  user_id:
+    type: string
+    description: the user that performed the delete
+  user_text:
+    type: string
+    description: the text representation of the user
+  summary:
+    type: string
+    description: the summary comment left by the user
+required:
+  - event
+  - title
+  - revision
```

Comment:
Since this is the first version of the schemas, I'd really like to nail down the minimalistic set on which we can all agree but still allow us to continue our work on change propagation. In that light, I don't see *namespace* to be an essential piece of information when deleting pages.